frontend/src/stores/OptionsStore.js

```diff
@@ -43,6 +43,10 @@ class OptionsStore {
     @computed get canAddNewOption() {
         return this.optionsList.length < 3;
     }
+
+    @computed get hasLogNormalSelected() {
+        return this.optionsList.filter(option => option.dist_type === 3).length > 0;
```

Comment:
can you create a constant for `DistTypeLognormal=3`  in `optionsConstants`? just so we're not passing ints around